airbyte-integrations/connectors/source-aha/acceptance-test-docker.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
 # Build latest connector image
-docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2-)
+docker build . -t "$(< acceptance-test-config.yml grep "connector_image" | head -n 1 | cut -d: -f2- | xargs)"
```

Comment:
Why are you doing those changes and changing the file permission from 100644 -> 100755?

Reply:
OK I'll revert that part - it's more capable with ShellCheck now, but as you wish